src/main/java/seedu/address/storage/Storage.java

```diff
@@ -29,4 +37,19 @@ public interface Storage extends AddressBookStorage, UserPrefsStorage {
     @Override
     void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException;
 
+    /**
+     * Saves the given {@link ObservableList} as a JSON file to the storage at filePathString.
+     * @param displayedList cannot be null.
+     * @param filePathString cannot be null.
+     * @throws IOException if there was any problem writing to the file.
+     */
+    static void saveDisplayedList(ObservableList<Person> displayedList, String filePathString) throws IOException {
```

Comment:
Is it okay to have a static method here?

Reply:
method name can be misleading as discussed